Signal/src/view controllers/CallViewController.swift

```diff
@@ -73,6 +73,12 @@ class CallViewController: UIViewController, CallObserver, CallServiceObserver, R
     var remoteVideoConstraints: [NSLayoutConstraint] = []
     var localVideoConstraints: [NSLayoutConstraint] = []
 
+    var areRemoteVideoControlsHidden = false {
```

Comment:
nit so that you know I actually read it: maybe `shouldRemoteVideoControlsBeHidden`